backend/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "spektate",
+  "version": "0.1.0",
+  "license": "MIT",
+  "dependencies": {
+    "@types/express": "^4.17.0",
+    "spektate": "^1.0.10",
+    "express": "^4.17.1"
+  },
+  "scripts": {
+    "start": "ts-node src/index.ts",
+    "build": "tsc src/** --outDir ../build/backend/ && tsc src/server.ts --outDir ../build/",
+    "build-prod": "tsc src/** --outDir ./build/",
+    "lint": "tslint -p tsconfig.json -c tslint.json src/*.ts"
+  },
+  "devDependencies": {
+    "@types/node": "^12.6.8",
+    "ts-node": "^8.5.4",
+    "tslint": "^5.20.0",
+    "tslint-config-prettier": "^1.18.0",
+    "tslint-react": "^4.2.0",
```

Comment:
do we need tslint-react for backend?

Reply:
will remove, it was added as part of `depcheck` i believe.